src/fastertransformer/models/deberta/DebertaWeight.cc

```diff
@@ -174,7 +174,7 @@ void DebertaWeight<T>::loadModel(std::string dir_path)
 
     for (uint l = 0; l < num_layer_; l++) {
         if (isValidLayerParallelId(l)) {
-            deberta_layer_weights[l].loadModel(dir_path + "model.encoder.layer." + std::to_string(l) + ".",
+            deberta_layer_weights[l].loadModel(dir_path + "/model.encoder.layer." + std::to_string(l) + ".",
```

Comment:
Is this a bug fix? Was this not being exercised in tests or samples before?

Reply:
yeah, it's a bug fix. It can be avoided by appending `/` in some var settings, but should fix it anyway.